.github/workflows/integrate.yaml

```
@@ -85,6 +85,8 @@ jobs:
         cp -r canonical-osm/charms/interfaces/juju-relation-mysql mysql
         sg microk8s -c 'juju bootstrap microk8s uk8s'
         juju add-model kubeflow
+        # Katib requires the namespace to have certain labels for metrics collection
+        sg microk8s -c 'kubectl apply -f https://raw.githubusercontent.com/kubeflow/katib/master/manifests/v1beta1/components/namespace/namespace.yaml'
```

I'm worried that if kubeflow/katib@master ever changes, our tests will break.  We could insource this file into the present repo, but I think this is an equivalent:
```suggestion
        sg microk8s -c 'kubectl label namespace kubeflow katib.kubeflow.org/metrics-collector-injection=enabled
```

---

I decided to add the labels on the default namespace and apply the example there to avoid messing with the juju model.